app/views/layouts/application.html.erb

```diff
@@ -38,6 +38,15 @@
   <body class="<%= logged_in? ? 'logged-in' : 'logged-out' %>" <%= PhraseApp::InContextEditor.enabled? ? "data-no-turbolink" : "" %>>
     <div id="outer" class="wrapper">
       <ul id="skiplinks"><li><a href="#main"><%= ts('Main Content') %></a></li></ul>
+      <%= content_for :footer_js do %>
```

Comment:
I put the code here because it's where the JavaScript will insert the banner, but it could've gone anywhere because it's really just inserting JavaScript in the footer.

Reply:
In that case, can we skip `content_for` and put the block below `<%= render :partial => 'layouts/javascripts' %>`?

Reply:
Sure, I can put a comment here instead.